apps/src/netsim/NetSimSendPanel.js

```diff
@@ -330,16 +345,19 @@ NetSimSendPanel.prototype.resetPackets_ = function () {
 NetSimSendPanel.prototype.onContentChange_ = function () {
   var nextBit = this.getNextBit_();
 
-  // Special case: If we have the "A/B" encoding enabled but _not_ "Binary",
-  // format this button label using the "A/B" convention
-  if (this.isEncodingEnabled_(EncodingType.A_AND_B) &&
-      !this.isEncodingEnabled_(EncodingType.BINARY)) {
-    nextBit = binaryToAB(nextBit);
-  }
+  if (nextBit === undefined) {
+    // If there are no bits queued up, disable the button
+    this.conditionallyToggleSetWireButton().text(i18n.setWire());
+  } else {
+    // Special case: If we have the "A/B" encoding enabled but _not_ "Binary",
+    // format this button label using the "A/B" convention
+    if (this.isEncodingEnabled_(EncodingType.A_AND_B) &&
+        !this.isEncodingEnabled_(EncodingType.BINARY)) {
+      nextBit = binaryToAB(nextBit);
+    }
 
-  this.getBody()
-      .find('#set-wire-button')
-      .text(i18n.setWireToValue({ value: nextBit }));
+    this.conditionallyToggleSetWireButton().text(i18n.setWireToValue({ value: nextBit }));
```

Comment:
I definitely far prefer the first suggestion, I'll do that now.

`toggle` is not setting the text because it's also called by the `.setSimplexWireState` continuation after the bit is consumed and everything is reenabled